auth/server/src/main/java/com/seagates3/parameter/validator/AccountParameterValidator.java

```diff
@@ -40,12 +40,30 @@
      */
     @Override
     public Boolean isValidCreateParams(Map<String, String> requestBody) {
-        if (!S3ParameterValidatorUtil.isValidEmail(requestBody.get("Email"))) {
-            return false;
+        int validParamsCount = 0;
+        if (S3ParameterValidatorUtil.isValidEmail(
+                requestBody.get("Email"))) {
+            validParamsCount++;
         }
 
-        return S3ParameterValidatorUtil.isValidName(
-                requestBody.get("AccountName"));
+        if(S3ParameterValidatorUtil.isValidName(
+                requestBody.get("AccountName"))) {
+            validParamsCount++;
```

Comment:
Why we are not returning like before. Why we have taken count?
To avoid multiple return statements?

Reply:
I took count for two reasons:

1. Yes to avoid multiple return statements. This makes readability better.
2. Easy to make complex validations. Or it will if () { return true } else { return false}

Also we can update the count as anyways we will be updating the method.

Reply:
This is hardcoding so not looking good here
You can take 1 flag instead of count.
eg- isValidationFailed = False;
As soon as any validation fails, you will set the flag to True.
You will include 1 more condition as follow-
if(!isValidationFailed  && S3ParameterValidatorUtil.isValidCustomAccessKey(accessKey))
and then have a single return statement at the end.

Reply:
Will fix as above

Reply:
Done